examples/traefik/plugin.yaml

```
@@ -0,0 +1,50 @@
+# Plugin Definition below. This is essentially a valid helm values file that will be merged
+# with the other vcluster values during vcluster create or helm install.
+plugin:
+  generic-crd-plugin:
+    image: ghcr.io/loft-sh/vcluster-generic-crd-plugin:latest
+    imagePullPolicy: Always
```

```suggestion
    imagePullPolicy: IfNotPresent
```
FYI, we changed all plugin examples to use `imagePullPolicy: IfNotPresent` because latest tag + imagePullPolicy: Always turned out to be a bad idea - if we were to build a version with a breaking change, or a bug, it might break some clusters at unexpected time.